provider/cmd/pulumi-tfgen-vault/main.go

```diff
@@ -17,8 +17,8 @@ package main
 import (
 	"github.com/pulumi/pulumi-terraform-bridge/pkg/tfgen"
 
-	vault "github.com/pulumi/pulumi-vault"
-	"github.com/pulumi/pulumi-vault/pkg/version"
+	vault "github.com/pulumi/pulumi-vault/provider"
+	"github.com/pulumi/pulumi-vault/provider/pkg/version"
```

Comment:
I do sort of wonder whether we should call this `provider` or `plugin`?  Thoughts @pgavlin?

Reply:
Either of those is fine, that's easy to change with no effect downstream.